test/seqan3-test.cmake

```diff
@@ -40,7 +40,12 @@ file(MAKE_DIRECTORY ${SEQAN3_TEST_CLONE_DIR}/googletest/include/)
 add_library (seqan3_test INTERFACE)
 target_compile_options (seqan3_test INTERFACE "-pedantic"  "-Wall" "-Wextra" "-Werror" "-DNO_WARN_X86_INTRINSICS")
 target_link_libraries (seqan3_test INTERFACE "seqan3::seqan3" "pthread")
-target_include_directories (seqan3_test INTERFACE "${SEQAN3_CLONE_DIR}/test/include/")
+if (SEQAN3_CLONE_DIR)
+    target_include_directories (seqan3_test INTERFACE "${SEQAN3_CLONE_DIR}/test/include/")
+else ()
+    target_include_directories (seqan3_test INTERFACE "${CMAKE_CURRENT_LIST_DIR}/include/")
+endif ()
```

Comment:
If `SEQAN3_CLONE_DIR/test == CMAKE_CURRENT_LIST_DIR` the second version would suffice.

Reply:
Do you still know why we needed this? Is the file structure in debian differently?

Reply:
Yes, In Debian we install the tests for running afterwards